misk/src/main/kotlin/misk/web/jetty/JettyService.kt

```diff
@@ -257,9 +263,11 @@ class JettyService @Inject internal constructor(
       gzipHandler.addExcludedMimeTypes(MediaTypes.APPLICATION_GRPC)
     } else {
       // GET is enabled by default for gzipHandler.
-      gzipHandler.addExcludedMethods("GET")
+      gzipHandler.addExcludedMethods("GET", "POST")
```

Comment:
This change was needed to return unzip POST response to support this test https://github.com/cashapp/misk/blob/master/misk/src/test/kotlin/misk/web/GzipTest.kt#L48-L49